crates/hyperswitch_domain_models/src/payments.rs

```diff
@@ -129,21 +132,21 @@ impl From<Option<bool>> for SurchargeCalculationOverride {
 #[derive(Clone, Debug, PartialEq, serde::Serialize)]
 pub struct AmountDetails {
     /// The amount of the order in the lowest denomination of currency
-    order_amount: MinorUnit,
+    pub order_amount: MinorUnit,
     /// The currency of the order
-    currency: common_enums::Currency,
+    pub currency: common_enums::Currency,
     /// The shipping cost of the order. This has to be collected from the merchant
-    shipping_cost: Option<MinorUnit>,
+    pub shipping_cost: Option<MinorUnit>,
     /// Tax details related to the order. This will be calculated by the external tax provider
-    tax_details: Option<TaxDetails>,
+    pub tax_details: Option<TaxDetails>,
```

Comment:
In api_model we have `order_tax_amount` and in domain_model we have `tax_details`, lets keep one name to denote these.

Reply:
In api_models, we'll be having order_tax_amount only where as in domain models we have default and pmt tax